modules/custom-status/custom-status.php

```diff
@@ -492,6 +492,16 @@ function update_custom_status( $status_id, $args = array() ) {
 		// Reset our internal object cache
 		$this->custom_statuses_cache = array();
 
+		// Prevent user from changing draft name or slug
+		if ( 'draft' === $old_status->slug
+		     && (
+			     ( isset( $args['name'] ) && $args['name'] !== $old_status->name )
+			     ||
+			     ( isset( $args['slug'] ) && $args['slug'] !== $old_status->slug )
+		     ) ) {
+			return new WP_Error( 'invalid', __( 'Changing the name and slug of "Draft" is not allowed', 'edit-flow' ) );
```

Comment:
As a user I think I might end up wondering why this isn't allowed. Perhaps we should put a link to some documentation or at least to the Git Hub issue that talks about it?

Reply:
Unless you "hack the system" you won't see that error. I've disabled input fields for drafts so that you can't actually change the slug and draft title.

Reply:
>Unless you "hack the system" you won't see that error. I've disabled input fields for drafts so that you can't actually change the slug and draft title.
Ah, I think I was confused, I thought this would impact current folks who changed it, but it's only when trying to change it.